src/libOpenImageIO/maketexture.cpp

```diff
@@ -62,6 +62,7 @@
   using boost::regex_replace;
 #else
 # include <regex>
+#include <vector>
```

Comment:
Are you sure you want this line here? This is a close that is specific to using regex in certain compiler combinations.

Reply:
No idea from where it comes. It just appeared with the last merge.
I will remove it!